src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -606,6 +629,35 @@ private static bool TryParseStatusCode(string responseHeader, int statusCodeStar
             return true;
         }
 
+        private static void HandleRedirectLocationHeader(EasyRequest state, string locationValue)
+        {
+            Debug.Assert(state._isRedirect);
+            Debug.Assert(state._handler.AutomaticRedirection);
+
+            string location = locationValue.Trim();
+            //only for absolute redirects
+            if (location.StartsWith(UriSchemeHttp) || location.StartsWith(UriSchemeHttps))
+            {
+                Uri forwardUri = new Uri(location);
```

Comment:
.Net and corefx windows versions are not throwing URL-format exceptions. So I am now using ```Uri.TryCreate```.